_usage/limitations-of-babelfish.md

```diff
@@ -302,21 +300,76 @@ features are discovered, and additional features are added to MS SQL:
 | `WITHOUT SCHEMABINDING` clause | Not supported in functions, procedures, triggers, or views. The object will be created, but as if `WITH SCHEMABINDING` was specified. |
 | `CREATE/ALTER/DROP SELECTIVE XML INDEX` clause | This syntax is not supported. |
 | `CREATE/ALTER/DROP XML SCHEMA COLLECTION` | This syntax is not supported. |
 
-However, more is missing than just features. There are also some
-corner cases which will provide the end user with [different
-behavior](/docs/usage/missing-features) than otherwise expected from MS SQL. It is
-important to understand those corner cases, as well.
+However, more is different than just features.
+For example, backup and recovery work differently in SQL Server and PostgreSQL,
+and you will have to use the PostgreSQL tools to backup Babelfish.
+It is important to understand such differences as well.
+
+## Dealing with limitations
+
+In this section, we've written a lot about limitations, unsupported features and
+such. However, how do you deal with these things in real life? If Babelfish
+runs a SQL statement that doesn't return a result set that is identical to the
+result set when run against SQL Server, the SQL statement will fail.
+To better deal with statements that might fail, Babelfish allows you to define
+*escape hatches*. An escape hatch is a flag that specifies Babelfish behavior
+when it encounters an unsupported feature or syntax. Babelfish can either return
+an error or ignore the condition if the PostgreSQL result set is not a perfect
+match for the result set returned by SQL Server.
+
+You can use the `sp_babelfish_configure` stored procedure to display or change
+the settings of each escape hatch. Use the script to specify if each escape
+hatch should be set to `ignored` or `strict`.
+
+If set to `strict`, Babelfish will return an error that you must
+correct before continuing. Include the `server` keyword to apply
+the changes to the current session as well as on a cluster level.
+
+The following escape hatches exist:
 
+- `storage_options`: Controls treatment of
+	- Column options (sparse files, file streams, `ROWGUIDCOL`)
+	- Index options (`PAD_INDEX`, `FILLFACTOR`, `SORT_IN_TEMPDB`, `IGNORE_DUP_KEY`,
+	  `STATISTICS_NORECOMPUTE`, `STATISTICS_INCREMENTAL`, `ALLOW_ROW_LOCKS`,
+	  `ALLOW_PAGE_LOCKS`, `OPTIMIZE_FOR_SEQUENTIAL_KEY`, `MAXDOP`, `DATA_COMPRESSION`)
+- `storage_on_partition`: Handle invalid partition streams
+- `database_misc_options`: Handle database chaining, database property
+  `TRUSTWORTHY`, databases with persistent log buffers
+- `language_non_english`: Create databases with default languages other than
+  English
+- `compatibility_level`: Currently no options.
+- `fulltext`: Databases with default fulltext languages. `CREATE FULLTEXT INDEX`,
+  `ALTER FULL TEXT INDEX`, `DROP FULL TEXT INDEX`
+- `schemabinding_function`: Handles errors when no `SCHEMABINDING` option is given
+- `schemabinding_trigger`: Throw an error when it is not given
```

Comment:
Too many referents: what is "it"? in the phrase, "Throw an error when it is not given" - pls specify for this and the following two lines

Reply:
I extended the documentation of all "schemabinding" escape hatches.